services/meta/data.go

```diff
@@ -37,6 +37,8 @@ type Data struct {
 
 	MaxShardGroupID uint64
 	MaxShardID      uint64
+
+	DefaultRetentionPolicyName string `json:"-"`
```

Comment:
I'm looking at this again and I can't find where JSON is ever used to serialize this structure. The open source version seems to use protobuf for this too. This might be worth removing.